test/Puzzle-test.js

```diff
@@ -1,3 +1,40 @@
 const chai = require('chai');
 const expect = chai.expect;
-const Puzzle = require('../js/Puzzle.js');
\ No newline at end of file
+const Puzzle = require('../js/Puzzle.js');
+
+describe('Puzzle', function () {
+  it('should check to see if the puzzle has been completed', function () {
+    var puzzle = new Puzzle();
+    expect(puzzle.puzzleCompleted).to.equal(false);
+    expect(puzzle.puzzleCompleted).to.equal(true);
+  });
+
+  it('should assign a puzzle randomly upon instantiation', function() {
+    var puzzle = new Puzzle(1);
+    expect(puzzle.currentPuzzle).to.not.equal(null);
```

Comment:
Probably not the most useful assertion here. This test is also describing a default property so it should go into a single it block that tests all default properties. You can just check that the `currentPuzzle` value is an Object (or whatever data type it should be) rather than checking that it's *not* null. In your code, you never seem to assign the currentPuzzle to a value of `null`, so this assertion isn't really doing much for you.